.github/workflows/ci.yml

```
@@ -550,6 +551,53 @@ jobs:
           DEFAULT_BRANCH: ${{ needs.build-info.outputs.default-branch }}
           RUFF_FORMAT: "github"
 
+  # Runs static checks for groups of files in the repository in a single process without passing .
+  # List of files
+  mypy:
+    timeout-minutes: 45
+    name: "MyPy checks for {{ matrix.mypy-packages }}"
```

Okay, so my las suggestion made it worse :-) - Placeholder is added by Github as suffix...
```suggestion
    name: "MyPy checks"
```

---

No no - it's ok. It's only until prerequisites are met - then they are expanded. All is good. Similar as in case of dynamic task mapping - same principle.

---

Ah... right... Matrix will add the parameters automatically in this case. But you can override it as well. I simply forgot to add `$` before `{{` 🤦